bizdevbible.md

```diff
@@ -0,0 +1,259 @@
+
+One Core on the Polkadot Cloud can execute 1,500 transactions per second, which is approximately 4 billion transactions per month. We have seen applications easily execute nearly 10 million transactions in a month without an issue.
+
+The Polkadot Cloud will be the best place to onboard:
+- Existing businesses who want to migrate existing large volumes of traffic.
+- Custom Web3 solutions, which gain access to low level building primitives.
+	- Privacy Chains
+		- Trusted Execution Environments (TEEs)
+		- Zero-Knowledge Systems
+	- Large State Chains
+		- Large File Storage (ipfs, filecoin, large individual items)
+		- Large State Trees (billions of items)
+	- Social Networks
+		- Feeless transactions
+		- Subsidized operations and usage
+		- Businesses looking to abstract the blockchain from their users.
+	- Low Latency Applications
+		- Gaming
+	- MEV Resistance / Control Over Transaction Ordering
+		- Specific kinds of DeFi
+	- etc..
+
+The Polkadot Cloud will not be the best place for:
+- New businesses and trying to grow a new customer base.
+- New developers as the minimum bar for entry is quite high relatively speaking.
+- End-users, as the Polkadot Cloud does not offer many services which appeal to this audience.
+
+These kinds of users should to the Polkadot Hub.
+
+### Polkadot Hub
+
+The Polkadot Hub will be a near-zero-cost entry into the Polkadot Ecosystem.
+
+Like many other blockchain platforms, the Polkadot Hub is a Layer 1 Smart Contract Network, secured by the Polkadot Cloud.
+
+The Polkadot Hub uses a next-generation smart contract platform that is backwards compatible with Solidity smart contracts.
+
+The Polkadot Hub also has Polkadot native features such as:
+- Staking
+- Governance
+- Treasury
+- Stablecoins
+- etc...
+
+These should appeal to non-developer audiences in addition to developers passionate about the Web3 space.
+
+The Polkadot Hub will be the best place to onboard:
+- Low Time Investment Builders
+	- New Developers
+	- Tinkerers
+	- Explorers (i.e. experts from other ecosystems)
+- Short-Term Projects
+	- Hackathons
+	- Fundraising
+	- Executing Legal Contracts
+- Low Overhead / Small Teams
+	- Meme coins
+	- Whitepaper deployment
+- Passionate Users
+	- Active Polkadot Community
+	- Active Users of Polkadot Hub Applications
+	- Stablecoin Users
+- Polkadot Native Extensions
+	- Account Abstractions
+	- DAOs
+	- Staking Pools
+
+The Polkadot Hub will not be the best place for:
+- Doing things at large scale
+	- Large volume of traffic
+		- Potential issues with transaction inclusion
+		- Spikes in fees
+	- Large volume of data
+		- Expensive for the business and/or the end users
+- Those who need more control over their application
+	- See Polkadot Cloud for customizability options.
+
+These kinds of users should go to the Polkadot Cloud.
+
+## Audiences
+
+TODO
+
+### Existing Web2 Businesses
+
```

Comment:
Quick outline points:
- The only blockchain that can actually scale to support real businesses and applications
- Focus on your business, let Polkadot provide infra/explorers/interop/upgrades
- Something about abstraction so their customers don't need to deal with all of the technicalities of blockchains

Reply:
I think i would want first some feedback these audiences are right and capture the groups we want to think about.

The comments here are good, but would be wasted if the audience would expand or retract in scope, or change altogether.